web/src/main/webapp/WEB-INF/pages/application_state.jsp

```diff
@@ -112,7 +113,7 @@
                           <div class="col-sm-8">
                               <p class="form-control-static ${stat.first ? 'stateData' : ''}"><c:out value = "${data.position.name}"/></p>
                               <c:if test="${stat.first}">
-                                  <input class="stateInput hidden" type="text" name="position.name" id="positionNameInput" value="<c:out value = "${data.position.name}"/>"
+                                  <input class="stateInput hidden" type="text" name="position.name" id="positionNameInput" value="${data.position.name}"
```

Comment:
The `<c:out>` tag was introduced to fix XSS vulnerability, as we discussed we should use `fn:escapeXML` in this situation.